__tests__/app.test.js

```diff
@@ -131,3 +131,63 @@ describe("Get 200: /api/reviews", () => {
       });
   });
 });
+
+describe("/api/reviews/:review_id/comments", () => {
+  test("Get 200: comments should be served with the most recent comments first", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        expect(comments).toHaveLength(3);
+        expect(comments).toBeInstanceOf(Array);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            review_id: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+          });
+        });
+      });
+  });
+  test("Get 200: should respond with an array of comments for given review_id with all properties", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        expect(comments).toHaveLength(3);
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("Get 400: should return bad request when id not valid", () => {
+    return request(app)
+      .get("/api/reviews/not-number/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("400: bad request");
+      });
+  });
+  test("Get 404: should return not found review id is number but doesnt exist", () => {
+    return request(app)
+      .get("/api/reviews/999/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("not found");
```

Comment:
the above `msg` includes the status code in the string, whereas this one doesn't. Personally, I prefer it without the status code in the string, but whichever you prefer, aim to be consistent with it.